report_stock_spreadsheet/.travis.yml

```diff
@@ -0,0 +1,32 @@
+sudo: true 
```

Comment:
@miguelchuga 
You need modify the .travis.yml from root path.
This .travis.yml there is in your module then is unused